Add events webhook docs (#6151)
wazuh/wazuh-documentation: source/user-manual/api/getting-started.rst

      "error": 0
    }

Ingest security events
^^^^^^^^^^^^^^^^^^^^^^

> 

Reviewer: Should we add a `.. versionadded:: 4.5.0` tag or is it unnecessary?

Author: That is relevant, I will add it.

Reviewer 2: @nico-stefani , `4.5` or `4.6`? Due to latest version changes, there's a mismatch between branch and `versionadded` tag here.

Author: @javimed, yes, you are right. Here is the fix 5f24621eb71820637fcab02a1b0bc1477eaad87e

      "error": 0
    }

Ingest security events
^^^^^^^^^^^^^^^^^^^^^^

.. versionadded:: 4.5.0

> It is possible to send security events to analyze using the Wazuh API directly.

Reviewer 2: ```suggestion
You can send security events for analysis using the Wazuh API.
```

.. code-block:: console

>     # curl -k -X GET 'https://localhost:55000/events' -H  'Authorization: Bearer $TOKEN' -H  'Content-Type: application/json' -d '{"events": ["Event value 1", "{\"someKey\": \"Event value 2\"}"]}'

Reviewer 3: ```suggestion
    # curl -k -X POST "https://localhost:55000/events" -H  "Authorization: Bearer $TOKEN" -H  "Content-Type: application/json" -d '{"events": ["Event value 1", "{\"someKey\": \"Event value 2\"}"]}'
```
The method was wrong. Also, when using `'` instead of `"` to wrap the Atuh token, I get this error:
```
# curl -k -X POST 'https://localhost:55000/agents' -H 'Authorization: Bearer $TOKEN'
{"title": "Unauthorized", "detail": "Invalid token"}
```

```
# curl -k -X GET "https://localhost:55000/agents" -H "Authorization: Bearer $TOKEN"
{"data": {"affected_items": [{"os": {"arch": "x86_64", "codename": "Focal Fossa", "major": "20", "minor": "04", "name":...
```

.. code-block:: json
    :class: output

    {
      "data": {
        "affected_items": [

        ],
        "total_affected_items": 2,
        "total_failed_items": 0,
        "failed_items": []
      },
      "message": "All events were forwarded to analisysd",
      "error": 0
    }

.. note::

>     A maximum of 100 events can be sent per request.

Reviewer 2: ```suggestion
    You can send a maximum of 100 events per request.
```

      "error": 0
    }

Ingest security events
^^^^^^^^^^^^^^^^^^^^^^

.. versionadded:: 4.6.0

You can send security events for analysis using the Wazuh API.

.. code-block:: console

    # curl -k -X POST "https://localhost:55000/events" -H  "Authorization: Bearer $TOKEN" -H  "Content-Type: application/json" -d '{"events": ["Event value 1", "{\"someKey\": \"Event value 2\"}"]}'

.. code-block:: json
    :class: output

    {
      "data": {
        "affected_items": [

        ],
        "total_affected_items": 2,
        "total_failed_items": 0,
        "failed_items": []
      },
      "message": "All events were forwarded to analisysd",
      "error": 0
    }

.. note::

>    You can send a maximum of 100 events per request.

Reviewer 3: I just noticed that we are not documenting the 30 requests/minute rate limit for this endpoint. 

Maybe we should include in this note a notice that the endpoint is not meant to ingest large amounts of logs and is therefore limited to 30 requests per minute and 100 events per request. I would also specify the 30 requests/minute on the configuration page (`max_request_per_minute`):
- https://documentation.wazuh.com/current/user-manual/api/configuration.html#access
